CI/scripts/algorithm_names_map.dat

```diff
@@ -0,0 +1,52 @@
+# #############################################################################
+# Certifier Framework Encryption algorigthm names mapping
```

Comment:
NOTE to reviewers: This dat mapping file drives the string substitution.

We should discuss and re-arrange these items with correct names. I am trying to build a taxonomy of these encryption algorithms ... none of which I fully understand.

Reply:
@jlmucb suggested we standardize on the following:

- Enc_method: Encryption algorithms supported.
- Digest_method: Cryptographic hash algorithms supported.
- Integrity_method: Integrity protection algorithms, used in conjunction with specific encryption algorithms.